### PR TITLE
chore: ak repo explo

### PR DESCRIPTION
**Pull request by kouyoumart**
## Description

Mini branch where, while exploring the repo, I corrected a few things.
Very few changes, most of them being in the `example/` folder:
- changes in the example's Readme, explaining how to run the make command, while using the right env file
- changes in the `example.env` file, and `deployer.env` template file (comments on separate files, to avoid errors when loading env vars loading).
- changes in the `cloudbuild_cd.yaml` file (multi-line command, and single quotes added to the cron schedule specification).

## Related Issue

N/A

## Type of Change

- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [x] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🔐 Security fix
- [x] 📚 Examples / docs / tutorials / dependencies update
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I've read the [`CONTRIBUTING.md`](https://github.com/artefactory/vertex-pipelines-deployer/blob/develop/CONTRIBUTING.md) guide.
- [x] I've updated the code style using `make format-code`.
- [ ] I've written tests for all new methods and classes that I created.
- [ ] I've written the docstring in Google format for all the methods and classes that I used.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.

### CODE REVIEW

File: example/README.md
Comment (julesbertrand) [inline example/README.md:1]: Can you check this is included and in line with what's in the main mkdocs documentation please ?

File: example/vertex/deployment/cloudbuild_cd.yaml
Comment (julesbertrand) [inline example/vertex/deployment/cloudbuild_cd.yaml:1]: Is this working finally ?

File: example/README.md
Comment (kouyoumart) [reply example/README.md:1]: Sounds good, will modify the `example.md` in the `docs/`

File: example/vertex/deployment/cloudbuild_cd.yaml
Comment (kouyoumart) [reply example/vertex/deployment/cloudbuild_cd.yaml:1]: yes, tested it to deploy the scheduled pipeline, using the Makefile, and it now works well.

File: example/README.md
Comment (kouyoumart) [reply example/README.md:1]: @julesbertrand 

As the example.md is [not yet deployed](https://github.com/artefactory/vertex-pipelines-deployer/blob/develop/mkdocs.yml), I propose to:

- still add those new changes to the example Readme
- not modify the [example.md](https://github.com/artefactory/vertex-pipelines-deployer/blob/develop/docs/example.md) now (as I might not have time to finish by Friday).
- Open a new branch to document this markdown file (`example.md`), in line with the examples folder Readme.

**Review by julesbertrand — APPROVED** [review]

**Pull request merged**